Runtime/JSNativeApi.cs

```diff
@@ -114,6 +114,13 @@ public static unsafe bool IsUndefined(this JSValue value)
     public static unsafe bool IsNull(this JSValue value)
         => value.TypeOf() == JSValueType.Null;
 
+    public static unsafe bool IsNullOrUndefined(this JSValue value) => value.TypeOf() switch
```

Comment:
>IsNullOrUndefined [](http://example.com/codeflow?start=30&length=17)

In JavaScript it is usually called 'Nullish': [https://developer.mozilla.org/en-US/docs/Glossary/Nullish](https://developer.mozilla.org/en-US/docs/Glossary/Nullish)

Reply:
Sometimes, but that term may be unfamiliar to many C# developers, or they may be confused about whether an empty string or other "falsy" values are also "nullish". So I prefer the more explicit name.